test/jdk/java/net/DatagramSocket/DatagramSocketMulticasting.java

```diff
@@ -132,13 +143,13 @@ static void testJoinGroup2(ProtocolFamily family,
         System.out.format("testJoinGroup2: local socket address: %s%n", s.getLocalSocketAddress());
 
         // check network interface not set
-        assertTrue(s.getOption(IP_MULTICAST_IF) == null);
+        assertEquals(s.getOption(IP_MULTICAST_IF), null);
```

Comment:
It's possible to use assertNull and assertNoNull if you want, it doesn't matter of course.

Reply:
That would probably be cleaner, I'll make that change